src/client-apps/nuxeo-drive.md

```diff
@@ -141,695 +141,12 @@ When a new version of Nuxeo Drive is available, a message is displayed at the bo
 
 1.  Click on the upgrade message at the bottom of the systray menu.
     ![]({{file name='drive-notif-upgrade-available.png' page='nuxeo-drive'}})
-2.  If you want Nuxeo Drive to update silently the next time, check the box **Automatically update Drive** in the **General** tab.
-    ![]({{file name='drive-tab-general.png' page='nuxeo-drive'}})
+2.  If you want Nuxeo Drive to update silently the next time, check the box **Install new version automatically** in the **Advanced** tab.
+    ![]({{file name='drive-tab-advanced.png' page='nuxeo-drive'}})
 3.  Click on the green icon.
     Nuxeo Drive is updated and automatically restarted.
     ![]({{file name='drive-notif-upgrade-progress.png' page='nuxeo-drive'}} ?w=300)
 
-## Configuration
```

Comment:
Those changes are super interesting.
I am worried about all links we already shared to customers (from support tickets mostly) that are pointing to those sections. Is it possible to add redirections links (I am not sure at all because it involves anchors, but it is worth asking)?

Reply:
Yes I will put redirects, all the links will still be working correctly.